app-modules/integration-twilio/database/migrations/2023_11_14_192817_create_twilio_settings.php

```diff
@@ -41,7 +41,7 @@
     public function up(): void
     {
         $this->migrator->inGroup('twilio', function (SettingsBlueprint $blueprint): void {
-            $blueprint->add('is_enabled', true); //TODO: force enabled for now
+            $blueprint->add('is_enabled', true);
```

Comment:
We might want to change this to false just to avoid situations like the chat being enabled, but Twilio not being configured so it throws an error.

Reply:
I believe we default this to being `true` because if we don't something throws an error.

I would prefer to leave this for now and I created a ticket to look into it.